### DISCUSSION TITLE
Get torrent "Added" time from httprpc

### DISCUSSION THREAD

**Discussion by gashtal** [Novik/ruTorrent#2584] [Q&A] [answered]
I am using xmlrpc through ruTorrent's httprpc plugin to query torrent data through command line. rTorrent provide `d.timestamp.finished` and `d.timestamp.started` for querying the timestamp a specific torrent was started and finished. However, I am looking for a way to query the "added" timestamp of a torrent which is displayed under the "Added" column in ruTorrent. Does anyone have any idea how I can do this with xmlrpc?

**Comment by koblack** [comment, marked as answer]
try `xmlrpc localhost d.custom <infohash> addtime`

**Comment by gashtal** [comment]
Worked beautifully; thank you!
